eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/runner/AccessionReleaseJobLauncherCommandLineRunnerTest.java

```diff
@@ -210,16 +210,16 @@ private void runJobBAndCheckResume(JobInstance failingJobInstance) throws Except
     private void injectErrorIntoMongoClusteredVariantOperationCollection() throws Exception {
         Query query = query(where("_id").is("5c519550dcc3abf7ee9125af"));
         Update update = new Update();
-        // Intentionally inject error in entry#7 (contig, start sorted) in the Mongo collection
-        update.set("inactiveObjects.0.accession", "881171680jibberish");
+        // Intentionally inject error in the Mongo collection
+        update.set("inactiveObjects.0.start", "8421515jibberish");
```

Comment:
Since we are no longer parsing out entire operations object in Deprecation reader, this was changed to [trip up this part](https://github.com/sundarvenkata-EBI/eva-accession/blob/533d3894dfe76ac02aa14ae225428510d956da8e/eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/deprecated/DeprecatedVariantMongoReader.java#L114) and trigger an exception.